backend/logs_helper/views.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+import os
+from datetime import datetime, timezone
+
+import redis
+from logs_helper.constants import LogsHelperKeys
+from rest_framework import status, viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from utils.local_context import StateStore
+
+from .serializers import StoreLogMessagesSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class LogsHelperView(viewsets.ModelViewSet):
+    """Viewset to handle all Tool Studio prompt related API logics."""
+
+    r = redis.Redis(
+        host=os.environ.get("REDIS_HOST", "http://localhost"),
+        port=os.environ.get("REDIS_PORT", "6379"),
+        username=os.environ.get("REDIS_USER", ""),
+        password=os.environ.get("REDIS_PASSWORD", ""),
+    )
+
+    @action(detail=False, methods=["get"])
+    def get_logs(self):
+        # Extract the session ID
+        session_id: str = StateStore.get(LogsHelperKeys.LOG_EVENTS_ID)
```

Comment:
Can't we extract the session-id from the request ?

refer this Util function https://github.com/Zipstack/unstract/blob/main/backend/utils/user_session.py#L6

Reply:
Yes, we can get session-id from cookies. but I can't find any functions to get session-id from the Util function you refereed. Please correct me if im wrong

Reply:
Sorry, I referred to the wrong util.

@muhammad-ali-e  dont have such util function already?